common/src/auth/SharedAccessSignature.cs

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Security.Cryptography;
+using System.Text;
+using System.Net;
+
+namespace Microsoft.Azure.Devices.Common.Authorization
+{
+    public class SharedAccessSignature
+    {
+        protected string _shareAccessSignatureName;
```

Comment:
Why have backing fields and properties? Are the backing fields needed?

Reply:
It's actually not, I will remove those